internal/cli/organizations/apikeys/whitelist/create.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package whitelist
+
+import (
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/output"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const createTemplate = "Created new whitelist entry(s).\n"
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	apyKey string
+	ips    []string
+	cidrs  []string
+	store  store.OrganizationAPIKeyWhitelistCreator
+}
+
+func (opts *CreateOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *CreateOpts) newWhitelistAPIKeysReq() ([]*atlas.WhitelistAPIKeysReq, error) {
+	var whitelistRep []*atlas.WhitelistAPIKeysReq
```

Comment:
we know the final size, let's set it up so we don't need to resize the slice latter
`whitelistRep := make([]*atlas.WhitelistAPIKeysReq, 0, len(opts.ips) + len(opts.cidrs))`